Components/Toast/toast.html

```diff
@@ -73,15 +74,45 @@
 
         <main>
             <div class="content-container">
-                <h1>Toast</h1>
+                <h1>Toast / Snackbar</h1>
                 <p>
-                    Toasts will come here.
+                    Toasts / Snackbar are push notifications for the user.
                 </p>
+                <br>
+                <br>
+                <br>
+                <br>
+                <div class="toast-container">
+                    <div class="component-snackbar"> 
+                        <button onclick="showToast()" class="solid-secondary-btn">Show Snackbar</button>
```

Comment:
Instead of adding JS here, you can add it in a separate JS File.

Reply:
Done. Thanks for review 😄